tools/data_integrity/score_instrument.php

```diff
@@ -38,33 +38,47 @@
     );
 }
 
-/**
+/* -------------------------
  * HELP SCREEN
  * display and stop processing if action=help
+ * -------------------------
  */
-if (isset($argv[1]) && $argv[1] === 'help' || in_array('-h', $argv, true)){
+if (isset($argv[1]) && $argv[1] === 'help' || in_array('-h', $argv, true)) {
     showHelp();
 }
 
 if (empty($argv[1])
     || ($argv[1] !== 'all' && empty($argv[2]))
-    || ($argv[1] !== 'all' && $argv[2]=='one' && (empty($argv[3]) || empty($argv[4])))
+    || ($argv[1] !== 'all'
+    && $argv[2]=='one'
+    && (empty($argv[3]) || empty($argv[4])))
 ) {
     echo "\nERROR: Too few arguments passed. Check documentation and try again.";
     showHelp();
 }
 if (($argv[1] === 'all' && isset($argv[2]) && $argv[2] !== 'inprogress')
-    || ($argv[1] !== 'all' && !in_array($argv[2], array('all', 'one'), true))
-    || ($argv[1] !== 'all' && $argv[2] === 'all' && isset($argv[3]) && $argv[3] !== 'inprogress')
-    || ($argv[1] !== 'all' && $argv[2] === 'one' && isset($argv[5]) && $argv[5] !== 'inprogress')
+    || ($argv[1] !== 'all' && !in_array($argv[2], ['all', 'one'], true))
+    || ($argv[1] !== 'all'
+    && $argv[2] === 'all'
+    && isset($argv[3])
+    && $argv[3] !== 'inprogress')
+    || ($argv[1] !== 'all'
+    && $argv[2] === 'one'
+    && isset($argv[5])
+    && $argv[5] !== 'inprogress')
 ) {
-    echo "\nERROR: Too many or incorrect arguments passed. Check documentation and try again.";
+    echo "\n
```

Comment:
This changes the value. The string sent to the terminal was:

```
ERROR: Too many or incorrect arguments passed. Check documentation and try again.
```

now it's

```

Error: Too many or or incorrect arguments passed.
Check documentation and try again.
```

with an extra newline at the start and a new one in the middle of the message.